### PR TITLE
Rustfmt

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

File: rustfmt.toml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+wrap_match_arms = true
+struct_lit_multiline_style = "ForceMulti"
+wrap_comments = false
+normalise_comments = false
```
Comment (alexcrichton) [inline rustfmt.toml]: Perhaps it's best to use the standard and default style? That'd generally get the most bang-for-its-buck, right?

**Comment by Stebalien** [comment]
I removed everything from rustfmt.toml but the line disabling `normalize_comments` because that feature breaks glob.

**Comment by alexcrichton** [comment]
Out of curiosity, how does it break? Would it be possible to unbreak with a local annotation?

**Comment by Stebalien** [comment]
It was turning:
```
None => { /* not a directory, nothing more to find */}
```
Into:
```
None => { // not a directory, nothing more to find }
```

Anyways, I fixed it with whitespace and got rid of the config.

**Comment by blaenk** [comment]
I feel like this should be reported upstream, no? Seems like a bug.

**Comment by Stebalien** [comment]
@blaenk Yes, I'm just lazy... https://github.com/rust-lang-nursery/rustfmt/issues/784

**Comment by alexcrichton** [comment]
Thanks for the investigation!

**Pull request merged**